### ISSUE TITLE
structurize duplicate

### ISSUE THREAD

**Issue by Osterzopf** [ldtteam/minecolonies#3497]
### Minecolonies version

Version: every version after 0.10.183-ALPHA
Tried all alpha versions up to 0.10.188, all release and beta versions and some of the most recent ones

### Expected behavior

Testet with version 0.10.183-ALPHA
- Forge loads Minecolonies without any problems
- The structurize jar and meta files can be found in the \mods\memory_repo\com\structurize\structurize folder
- Even though there are two versions of the structurize jars and meta files (0.10.36-ALPHA and 0.10.47-ALPHA) this doesn't count as a duplicate

### Actual behaviour

Testet with version 0.10.202_RELEASE

- The normal forge loading screen get interrupted and Minecraft shows only this message:
![minecolonies](https://user-images.githubusercontent.com/48602697/54459452-00ac3480-4767-11e9-9ba8-3e13287bd895.PNG)
- The log shows a fatal error because of duplicate mods and throws an exception:
[21:26:04] [main/FATAL] [FML]: Found a duplicate mod structurize at [D:\Program Files\MultiMC\instances\Fantasy+Evolved+II-11.2\minecraft\mods\memory_repo\com\ldtteam\structurize\1.12.2-0.10.63-ALPHA\structurize-1.12.2-0.10.63-ALPHA.jar, D:\Program Files\MultiMC\instances\Fantasy+Evolved+II-11.2\minecraft\mods\memory_repo\com\structurize\structurize\1.12.2-0.10.36-ALPHA\structurize-1.12.2-0.10.36-ALPHA.jar]
[21:26:04] [main/ERROR] [FML]: An exception was thrown, the game will display an error screen and halt.
net.minecraftforge.fml.common.DuplicateModsFoundException: 
Duplicate Mods:
	structurize : D:\Program Files\MultiMC\instances\Fantasy+Evolved+II-11.2\minecraft\mods\memory_repo\com\ldtteam\structurize\1.12.2-0.10.63-ALPHA\structurize-1.12.2-0.10.63-ALPHA.jar
	structurize : D:\Program Files\MultiMC\instances\Fantasy+Evolved+II-11.2\minecraft\mods\memory_repo\com\structurize\structurize\1.12.2-0.10.36-ALPHA\structurize-1.12.2-0.10.36-ALPHA.jar

	at net.minecraftforge.fml.common.Loader.identifyDuplicates(Loader.java:464) ~[Loader.class:?]
	at net.minecraftforge.fml.common.Loader.identifyMods(Loader.java:426) ~[Loader.class:?]
	at net.minecraftforge.fml.common.Loader.loadMods(Loader.java:566) ~[Loader.class:?]
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:232) [FMLClientHandler.class:?]
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:466) [bib.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377) [bib.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:123) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_191]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_191]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_191]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_191]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_191]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_191]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_191]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_191]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[21:26:04] [main/WARN] [FML]: EventBus 0 shutting down - future events will not be posted.
- The two structurize versions are now split in different folders. One at \mods\memory_repo\com\ldtteam\structurize and the other one in the old \mods\memory_repo\com\structurize\structurize. But they can't be the same. They even have different sizes

### Steps to reproduce the problem

1. Install an "old" version (I used MultiMC)
2. Have fun playing the game
3. Install a "new" version instead
4. Get frustrated about the error
5. ... Profit

**Comment by Asherslab** [comment]
Hey there mate, it would seem you have downloaded our Fat jar, which is the first one that shows up in curseforge, if you look here: 
![Screen Shot 2019-03-15 at 4 29 30 pm](https://user-images.githubusercontent.com/28527281/54463064-d004d480-473f-11e9-89d8-b0da7dee8153.png) you can see the "minimal" jar provided below, while we would much rather you use the minimal jar and download the structurize jar separately, we will not be "forcing" this until 1.13. 
The fat jar contains structurize inside it, this is only being provided for now to avoid confusion

TLDR: you're using a fat jar, and you don't need to have the extra structurize jar as wel
